### PR TITLE
feat(route): cmpxchg8b

### PR DESCRIPTION
**Pull request by yuguorui**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/cmpxchg8b/articles
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [x] Documentation / 文档说明
  - [x] EN / 英文文档
  - [x] CN / 中文文档
- [x] Full text / 全文获取
  - [x] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制 
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施? 
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: website/docs/routes/programming.mdx [hunk 1]
```diff
@@ -166,6 +166,12 @@ Category
 
 <Route author="elxy" example="/bitmovin/blog" path="/bitmovin/blog"/>
 
+## cmpxchg8b {#cmpxchg8b}
+
+### articles {#cmpxchg8b-articles}
```
Comment (TonyRL) [inline website/docs/routes/programming.mdx:171]:
```suggestion
### Articles {#cmpxchg8b-articles}
```

File: lib/v2/cmpxchg8b/articles.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const cheerio = require('cheerio');
+
+const baseUrl = 'https://lock.cmpxchg8b.com/';
+const title = 'cmpxchg8b';
+
+module.exports = async (ctx) => {
+    const { data: response } = await got({
+        method: 'get',
+        url: baseUrl,
+    });
```
Comment (TonyRL) [inline lib/v2/cmpxchg8b/articles.js:11]: https://github.com/DIYgod/RSSHub/blob/31774e8b7cd8e5d38e5c4f05b87ba10c6081e9c7/lib/v2/cmpxchg8b/articles.js#L29
Please use either one style but not both.

File: lib/v2/cmpxchg8b/articles.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        });
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const { data: response } = await got(item.link);
+                const $ = cheerio.load(response);
+
+                // after #TOC but before footer
+                item.description = $('nav#TOC').nextUntil('footer').html();
```
Comment (TonyRL) [inline lib/v2/cmpxchg8b/articles.js:33]: This does not work in the latest 3 articles as you can see from the [preview](https://github.com/DIYgod/RSSHub/pull/13802#issuecomment-1812528568).

Comment (yuguorui) [reply lib/v2/cmpxchg8b/articles.js]: Thanks.

Fixed by removing unwanted elements from the body explicitly.

```js
  // extract the body but removing <nav> and <footer>
  const body = $('body');
  body.find('nav, footer').remove();
  item.description = body.html();
  return item;
```

**Pull request merged**